ci.gradle

```diff
@@ -6,7 +6,7 @@ task testReport(type: TestReport) {
     description = "Aggregate all test results from subprojects into one test report."
     group = "CI"
 
-    destinationDir = file("$buildDir/reports/allTests")
+    destinationDir = file("$buildDir/reports/tests")
```

Comment:
could we use the same path here and on l.27. Here you do `$buildDir` there it's `rootProject.buildDir`. I believe it will end up being the same path but I would still prefer these two to be in sync